.babelrc

```diff
@@ -3,6 +3,19 @@
     [
       "@babel/preset-env",
       {
+        "targets": [
+          "last 3 major versions",
+          "Firefox ESR",
+          "Chrome >= 53",
+          "not dead",
+          "not ie 11",
+          "not baidu 7",
+          "not and_qq 11",
+          "not and_uc 12",
+          "not kaios 2",
+          "not op_mini all",
+          "not op_mob 64"
+        ],
```

Comment:
Question: Should we specify the minimum Safari version here?

Reply:
I _think_ `last 3 major versions` covers Safari adequately.